crates/compilers/src/compilers/resolc/compiler.rs

```diff
@@ -126,84 +126,97 @@ impl Resolc {
         }
     }
 
-    pub fn install(resolc_version: Option<&Version>, solc_compiler: SolcCompiler) -> Result<Self> {
-        let solc_version = match &solc_compiler {
+    pub fn install(_resolc_version: Option<&Version>, solc_compiler: SolcCompiler) -> Result<Self> {
+        let _solc_version = match &solc_compiler {
             SolcCompiler::Specific(solc) => Some(solc.version_short()),
             #[cfg(feature = "svm-solc")]
             SolcCompiler::AutoDetect => None,
         };
-        let version_manager =
-            rvm::VersionManager::new(false).map_err(|e| SolcError::Message(e.to_string()))?;
+        #[cfg(any(feature = "async", feature = "svm-solc"))]
+        return foundry_compilers_core::utils::RuntimeOrHandle::new().block_on(async {
```

Comment:
I suppose it is related to this: https://github.com/seanmonstar/reqwest/issues/1017
Could you please add some comment?

Reply:
Could we merge to `main` instead of `revive`?

Reply:
added a comment